client/src/pages/donate-other.js

```diff
@@ -0,0 +1,264 @@
+/* eslint-disable max-len */
+import React, { Component, Fragment } from 'react';
+import Helmet from 'react-helmet';
+
+import Spacer from '../components/helpers/Spacer';
+
+class IndexPage extends Component {
+  render() {
+  return (
+    <Fragment>
+      <Helmet title='Other ways to donate | freeCodeCamp.org' />
+      <Spacer />
+      <div className='container'>
+        <div className='row'>
+          <div className='col-xs-12 col-sm-10 col-sm-offset-1 col-md-6 col-md-offset-3'>
+            <h2 className='text-center'>
+              Other ways you can support the freeCodeCamp.org nonprofit
+            </h2>
+            <p>
+              freeCodeCamp is a small donor-supported 501(c)(3) public charity. We
+              are tax-exempt, so you can deduct donations you make to our
+              nonprofit from your taxes. You can{' '}
+              <a href='https://s3.amazonaws.com/freecodecamp/Free+Code+Camp+Inc+IRS+Determination+Letter.pdf'>
+                download our IRS Determination Letter here
+              </a>.
+            </p>
+            <hr />
+            <h3>Set up a monthly donation using PayPal</h3>
+            <p>
+              You can set up a monthly donation to freeCodeCamp by clicking one of
+              the links below and following the instructions on PayPal. You can
+              easily stop your donations at any time in the future.
+            </p>
+            <form
+              action='//www.paypal.com/cgi-bin/webscr'
+              method='post'
+              onSubmit="ga('send',  { hitType: 'event', eventCategory: 'donation', eventAction: 'click', eventLabel: 'paypal', eventValue: 5 } );"
```

Comment:
Is there a way to refactor this to use our `ga` in `client/src/analytics`? It might even enable us to  refactor this whole list of buttons in to an array to map over.

Reply:
Yep. Will do once I address:

> Literal copy paste of the html version. This entire page will be further refactored to react components.

Reply:
OK, I thing GA will have to be what it is for now, until we overhaul the thing. Will have to come in a separate PR.